saleor/webhook/tests/test_tasks.py

```diff
@@ -1149,3 +1152,57 @@ def test_handle_transaction_request_task_request_event_included_in_calculations(
         timeout=mock.ANY,
         allow_redirects=False,
     )
+
+
+@mock.patch("saleor.webhook.tasks.send_webhooks_async_for_app")
+def test_process_async_webhooks_task(
+    mock_send_webhooks_for_app, app_with_webhook, event_payload
+):
+    # given
+    app_1, app_2, app_3, app_4 = App.objects.bulk_create(
+        [
+            App(name="App_1", is_active=True),
```

Comment:
What if app is inactive?

Reply:
I think we should process webhooks no matter the app is active or not. Otherwise we can end up with unprocessed webhooks in the queue in case someone deactivate the app just after generating some webhook. What do you think ?

Reply:
I discussed it with a product team - you are right, we should only process webhooks for active apps. Undelivered webhooks should wait for the app to be re-enabled or uninstalled